.github/workflows/docs.yml

```diff
@@ -8,66 +8,61 @@ on:
   workflow_dispatch: # run on request (no need for PR)
 
 permissions:
-  contents: write
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow one concurrent deployment
+concurrency:
+  group: "pages"
+  cancel-in-progress: true
 
 jobs:
   Build-and-Publish-Documentation:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
     runs-on: [docs]
     steps:
       - name: CHECKOUT REPOSITORY
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
           python-version: "3.10"
+
       - name: Install requirements
         run: |
           pip install -r requirements/docs.txt
-          pip install .[full]
+          pip install ".[full]"
+
       - name: Link dataset path to local directory as nbsphinx runs the notebook
         run: ln -s $ANOMALIB_DATASET_PATH ./datasets
       - name: Build and Commit Docs
         run: |
           cd docs
           make html
-      - name: Create gh-pages branch
-        run: |
-          echo ::set-output name=SOURCE_NAME::${GITHUB_REF#refs/*/}
-          echo ::set-output name=SOURCE_BRANCH::${GITHUB_REF#refs/heads/}
-          echo ::set-output name=SOURCE_TAG::${GITHUB_REF#refs/tags/}
+          cd ..
 
-          existed_in_remote=$(git ls-remote --heads origin gh-pages)
-
-          if [[ -z ${existed_in_remote} ]]; then
-            echo "Creating gh-pages branch"
-            git config --local user.email "action@github.com"
-            git config --local user.name "GitHub Action"
-            git checkout --orphan gh-pages
-            git reset --hard
-            touch .nojekyll
-            git add .nojekyll
-            git commit -m "Initializing gh-pages branch"
-            git push origin gh-pages
-            git checkout ${{steps.branch_name.outputs.SOURCE_NAME}}
-            echo "Created gh-pages branch"
-          else
-            echo "Branch gh-pages already exists"
-          fi
-      - name: Commit docs to gh-pages branch
+      - name: Clean directory
         run: |
-          git fetch
-          git checkout gh-pages
           mkdir -p /tmp/docs_build
-          cp -r docs/build/html/* /tmp/docs_build/
+          cp -r docs/build/html/* /tmp/docs_build
```

Comment:
Can you explain this change?

Reply:
Probably it was not present when I copied this command. I guess it should not affect copying the folder

Reply:
As far as I know `/` keeps the directory structure, while the latter just flattens the structure. If it works, I'm fine with it though